resource-managers/kubernetes/docker/src/main/dockerfiles/spark-base/Dockerfile

```diff
@@ -38,7 +38,7 @@ COPY jars /opt/spark/jars
 COPY bin /opt/spark/bin
 COPY sbin /opt/spark/sbin
 COPY conf /opt/spark/conf
-COPY dockerfiles/spark-base/entrypoint.sh /opt/
+COPY kubernetes/dockerfiles/spark-base/entrypoint.sh /opt/
```

Comment:
Do you mean this ? https://github.com/apache/spark/pull/19946/files#diff-cb1784535ec45c5b890bb16c5971be65R26

Reply:
Yes, to elaborate, on our fork, we used to have `dockerfiles/` in the root of the distribution. But now, it's copied to `kubernetes/dockerfiles` which is a more logical place, since there is a `yarn/` in the distribution as well.